pkgs/tools/system/incron/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, bash }:
+
+stdenv.mkDerivation rec {
+  name = "incron-0.5.12";
+  src = fetchurl {
```

Comment:
Why not use `fetchFromGitLab`?

Reply:
Generally speaking, is fetchFromGitLab/fetchFromGitHub preferred over fetchurl?

Reply:
Yes. There can be some better caching done or something.

Reply:
Thank you for explaining. I've adjusted as recommended. Thank you.